src/data/plugins.json

```diff
@@ -1206,6 +1206,13 @@
           "link": "https://github.com/mailslurp/cypress-mailslurp",
           "keywords": ["email", "mailslurp", "test", "commands"],
           "badge": "community"
+        },
+        {
+          "name": "cypress-mailhog",
+          "description": "A collection of usefull Cypress commands for testing Emails utilizing the MailHog RestAPI. Comes with TypeScript support.",
```

Comment:
Typo
usefull  > useful

Reply:
Thanks, just fixed the typo. 

Note: the `Suggestion` Feature would make it possible for me to right away apply the suggested changes. This is very useful for smaller stuff like typos 😊 

<img width="288" alt="image" src="https://user-images.githubusercontent.com/7868768/236311447-66f3db71-16ae-4706-8471-26f6ad709074.png">

Reply:
Sure. I use that also, but didn't this time.

Reply:
@SMenigat 
You also have this typo in your repo README.